participants/shayan

```diff
@@ -0,0 +1,50 @@
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Shayan",
+		"familyName": "ypn",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": false,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": true
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "yazdanpanahH",
+	// company name (optional)
+	"company": "ACME",
+	// either both days or at least one day has to be set to true.
+	// If you cannot make one of the days, please set it to false, to allow someone else take that spot!
+	"when": {
+		// June 27th, 2025
+		"friday": true,
+		// June 28th, 2025
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": true,
+	// your current interests (JS and in general) (required)
+	"tags": ["ReScript", "React", "Zustand", "GraphQL"],
```

Comment:
Are you sure about these tags? Feels like the defaults from the template favoring React over Svelte 😢 😅